assets/js/components/settings/SettingsCardConsentMode.js

```diff
@@ -103,22 +139,20 @@ export default function SettingsCardConsentMode() {
 					</Row>
 					{ ! isLoading && (
 						<Fragment>
-							{ consentAPIInfo?.hasConsentAPI &&
```

Comment:
This was missed in #8278 - we should show this CTA notice regardless of whether the WP Consent API is present. Fixing here rather than raising a separate followup PR.